flac.go

```diff
@@ -111,6 +130,33 @@ func (stream *Stream) parseStreamInfo() (isLast bool, err error) {
 	return block.IsLast, nil
 }
 
+// skipId3v2 skips ID3v2 data prepended to flac files.
+func (stream *Stream) skipId3v2() (err error) {
```

Comment:
Capitalize ID, I think golint may be upset otherwise: https://github.com/golang/lint/blob/a5f4a247366d8fc436941822e14fc3eac7727015/lint_test.go#L224

Mostly nit picks, but use named return values only when they add to documentation: https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters